misc/graylog.conf

```diff
@@ -139,9 +139,10 @@ plugin_dir = plugin
 # The maximum size of the HTTP request headers in bytes.
 #http_max_header_size = 8192
 
-# The size of the thread pool used exclusively for serving the HTTP interface.
+# The size of the thread pool used exclusively for serving the HTTP interface and REST API.
```

Comment:
FWIW, "HTTP interface" is not synonymous with "web interface".

Reply:
Aye, I misread that. :white_check_mark: